source/marc/terms.ttl

```diff
@@ -579,9 +583,22 @@ marc:city a owl:DatatypeProperty ;
 
 # 856:
 
+marc:versionOfResource a owl:ObjectProperty ;
+    rdfs:label "annan relaterad resurs"@sv ;
+    sdo:domainIncludes kbv:Item, kbv:Instance;
+    rdfs:range kbv:Electronic .
+
+marc:hostName a owl:DatatypeProperty ;
+    rdfs:label "Värddator (namn)"@sv ;
+    sdo:domainIncludes kbv:Document, kbv:MediaObject, kbv:Electronic .
+
+marc:electronicName a owl:DatatypeProperty ;
+    rdfs:label "Elektroniskt namn"@sv ;
+    sdo:domainIncludes kbv:Document, kbv:MediaObject, kbv:Electronic .
+
 marc:linkText a owl:DatatypeProperty ;
     rdfs:label "Länktext"@sv ;
-    rdfs:domain kbv:Note .
+    rdfs:domain kbv:Note, kbv:Document, kbv:MediaObject, kbv:Electronic .
```

Comment:
This needs to be domainIncludes (otherwise the domain is implied to be *all* of those).

Reply:
What a blunder. I'll fix.